_scripts/instruction-widget/install.js

```diff
@@ -107,8 +110,8 @@ module.exports = function(context) {
       context.certonly = true;
     }
 
-    // Debian Jessie backports.
-    if (context.version == 8) {
+    // Jessie backports.
+    if ((context.devuan && context.version == 1) || context.version == 8) {
```

Comment:
Is it true that on devuan 0 you'll not need to enable backports, but will be able to use apt to install it?